Sources/Data Model/Audience/Audience.swift

```diff
@@ -64,6 +64,21 @@ struct Audience: Codable, Equatable {
     }
     
     func evaluate(project: ProjectProtocol?, attributes: OptimizelyAttributes?) throws -> Bool {
-        return try conditions.evaluate(project: project, attributes: attributes)
+        let logger = OPTLoggerFactory.getLogger()
+        if let conditions = try? JSONEncoder().encode(conditions), let conditionsString = String(data: conditions, encoding: .utf8)  {
```

Comment:
Let's change this for efficiency (skipping whole parsing and string prepare for lower log-level)
```suggestion
        logger?.d {
            if let conditions = try? JSONEncoder().encode(conditions), let conditionsString = String(data: conditions, encoding: .utf8) {
                return LogMessage.audienceEvaluationStarted(id, conditionsString).description
            } else {
                return "Invalid conditions format"
            }
        }
```